CHANGELOG-hmp-198-data-transfer-no-public-globus.md

```diff
@@ -0,0 +1,2 @@
+- Added links to access dataset information in dbGaP.
+- Built out the UI for Protected Data when user is not logged in.
```

Comment:
The CHANGELOG-* content should read closer to Release Notes. The audience is someone who needs to know what has happened to the software or app and why. Commit messages are for communicating with folks involved in development and design and is a better forum for text like "Built out the UI for Protected Data when user is not logged in."

It may be helpful to review the content in the Portal UI CHANGELOG.md file which is generated from these CHANGELOG-*.md files:
- https://github.com/hubmapconsortium/portal-ui/blob/main/context/app/markdown/CHANGELOG.md
- https://github.com/hubmapconsortium/search-api/releases